### DISCUSSION TITLE
why network manager cant be restarted after sleep

### DISCUSSION THREAD

**Discussion by ghost** [davmac314/dinit#298] [Q&A]
I use artix with kde5. When ever I sleep or suspend my computer, the network manager services cant be restarted. How to resolve this?

**Comment by davmac314** [comment]
This is probably nothing to do with dinit really. Check the status (`dinitctl status NetworkManager`) and the system log, they might have a clue as to what would be going wrong.

**Reply by ghost** [reply]
sr for late reply,  **dinitctl status NetworkManager** did show NetworkManager was running, but in fact it's not. I cant stop or start it, but restart works. Not just NetworkManager, the system is hanged (the daemons and services went sleep and not able to wake them up by the system?). When "sudo reboot", the system seem go suspend and I have to manually hit the power button to start it up.

**Reply by davmac314** [reply]
> did show NetworkManager was running, but in fact it's not

How do you know it's not running?

> When "sudo reboot", the system seem go suspend and I have to manually hit the power button to start it up.

This sounds like a kernel issue.
